auth0/src/test/java/com/auth0/android/util/AuthenticationAPI.java

```diff
@@ -106,6 +106,12 @@ public AuthenticationAPI willReturnSuccessfulSignUp() {
         return this;
     }
 
+    public AuthenticationAPI willReturnSuccessfulEmptyBody() {
+        String json = "{}";
+        server.enqueue(responseWithJSON(json, 200));
```

Comment:
from the docs:

> If the request is valid, the refresh token is revoked and the response is HTTP 200, with an empty response body

empty response body is not the same as an empty object in the body.
just to be sure that when the test passes there are no issues when deserializing the response, let's make this return an actual empty body.